### PR TITLE
Community templates: include template contents when publishing

### PR DESCRIPTION
**Pull request by steveukx**
Include missing files from package content of community template packages.

### CODE REVIEW

**Comment by cristiano-belloni** [comment]
Tests are failing because of the extraneous changelog problem - `+ ├─ CHANGELOG.md`. I have an easy fix for it (5 lines or so). Do you want to remove the tests or me to add the fix to this PR?

File: packages/modular-template-app/package.json [hunk 1]
```diff
@@ -10,6 +10,8 @@
   },
   "license": "Apache-2.0",
   "files": [
-    "README.md"
+    "README.md",
```
[the remark this answers is not in the document]
Comment (steveukx) [reply packages/modular-template-app/package.json:13]: I'm going with light-touch so thought I'd leave these in the source as a reminder that we should be providing readme files

**Comment by steveukx** [comment]
> Tests are failing because of the extraneous changelog problem - `+ ├─ CHANGELOG.md`. I have an easy fix for it (5 lines or so). Do you want to remove the tests and PR later or do you want me to add the fix to this PR @steveukx ?

Please push to this branch @cristiano-belloni so we can pull it in a single merge, thanks

**Comment by cristiano-belloni** [comment]
@steveukx done

**Comment by steveukx** [comment]
✅ LGTM @cristiano-belloni

**Pull request merged**